src/ui/atoms/button-icon/state-props.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { ButtonIcon } from 'ui';
+import { IconSearch } from 'static/icons';
+
+import { GenerateMap, SizeBlock } from '../../../lib/map-generation';
```

Comment:
импорт из `lib/map-generation`

Reply:
```suggestion
import { GenerateMap, SizeBlock } from 'lib/map-generation';
```

Можно сразу подкинуть code suggestion, чтобы было проще